src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
I wasn't aware of popen(). Having the output show in the dosemu window was indeed something I hoped for to be possible. Thanks for the hint!